libafl/src/executors/combined.rs

```diff
@@ -24,9 +24,9 @@ impl<A, B> CombinedExecutor<A, B> {
     pub fn new<EM, Z>(primary: A, secondary: B) -> Self
     where
         A: Executor<EM, Z>,
-        B: Executor<EM, Z, State = A::State>,
-        EM: UsesState<State = A::State>,
-        Z: UsesState<State = A::State>,
+        B: Executor<EM, Z, State = <Self as UsesState>::State>,
```

Comment:
But that gets longer overall, do we really want that?

Reply:
Yes I would argue so.